lfucg-exactions/client/js/components/LotForm.js

```diff
@@ -532,7 +519,23 @@ function mapDispatchToProps(dispatch, params) {
                 dispatch(formUpdate(else_update));
             }
             dispatch(getLots());
-            dispatch(getPlats());
+            dispatch(getPlats())
+            .then((all_plats) => {
+                if (plat_start) {
+                    const plat_start_number = parseInt(plat_start, 10);
+                    const starting_plat = all_plats.response.filter(plat => (plat.id === plat_start_number));
+
+                    const plat_update = {
+                        plat: starting_plat[0].id,
```

Comment:
no .exists() check before accessing index?

Reply:
Seconded.  Since you're using starting_plat[0] multiple times, it probably makes sense to define it as its own variable before using it below.  Like so:

const matching_plats = all_plats.response.filter(...
if matching_plats.length > 0 {
    starting_plat = matching_plats[0]
    ... do stuff with starting_plat
}